helpers/deploy/constants.ts

```diff
@@ -32,6 +34,7 @@ export const LZ_CHAINID: Record<SUPPORTED_NETWORKS, number> = {
   opbnbtestnet: 10202,
   arbitrumsepolia: 10231,
   hardhat: 10102,
+  zksyncsepolia: 0, // TO-DO
```

Comment:
```suggestion
  zksyncsepolia: 10248,
```

According to https://docs.layerzero.network/v1/developers/evm/technical-reference/testnet/testnet-addresses#zksync-sepolia